src/main/java/reposense/git/GitBlame.java

```diff
@@ -57,6 +63,23 @@ public static String blameWithPreviousAuthors(String root, String fileDirectory)
         return StringsUtil.filterText(runCommand(rootPath, blameCommandWithFindingPreviousAuthors), COMBINATION_REGEX);
     }
 
+    /**
+     * Returns the processed git blame result, created from the raw git blame result.
+     */
+    public static List<GitBlameLineInfo> blameFile(String blameResults) {
+        String[] blameResultLines = StringsUtil.NEWLINE.split(blameResults);
+        List<GitBlameLineInfo> blameFileResult = new ArrayList<>();
+        for (int lineCount = 0; lineCount < blameResultLines.length; lineCount += BLAME_LINE_INFO_ROW_COUNT) {
+            String blameResultLine = Arrays.stream(Arrays
+                    .copyOfRange(blameResultLines, lineCount, lineCount + BLAME_LINE_INFO_ROW_COUNT - 1))
+                    .reduce("", (curr, next) -> curr + next + "\n");
```

Comment:
I am thinking of moving StringsUtil.NEWLINE.split(blameResult) to blameLine, so that this reduce function is no longer needed. What do you think?

Reply:
Sure, go ahead!